<launch>
    <node name="ActionServer" pkg="robot_maze_runner" type="ActionServer.py" output="screen"/>
    <node name="MazeRunner2" pkg="robot_maze_runner" type="MazeRunner2.py" output="screen"/>
</launch>
